core/src/main/java/org/elasticsearch/common/breaker/CircuitBreaker.java

```diff
@@ -32,6 +32,7 @@
     String FIELDDATA = "fielddata";
     String REQUEST = "request";
     String IN_FLIGHT_REQUESTS = "in_flight_requests";
+    String BUCKET = "bucket";
```

Comment:
I suppose one nice thing about having an bucket circuit breaker is that you **know** that buckets take up too much space. Without it you just know that buckets pushed it over the edge.